style.css

```diff
@@ -0,0 +1,635 @@
+* {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+  font-family: "Lato", sans-serif;
+  scroll-behavior: smooth;
+}
+
+header {
+  background-image: url(Images/viber_image_2022-12-28_12-03-05-282.jpg);
+  background-repeat: no-repeat;
+  background-size: 100%;
+}
+
+:root {
+  --black: #272a31;
+  --medium: #545454;
+  --light: #d3d3d3;
+  --white: #f9f9f9;
+  --dark-blue: #272a33;
+  --dark-background: #414246;
+  --orange: #ec5242;
+  --desktop-padding: 70px;
+}
+
+#first_page_about {
+  text-align: center;
+  background-image: url(Images/viber_image_2022-12-28_12-03-05-282.jpg);
+  background-repeat: no-repeat;
+  background-size: 100%;
+}
+
+.see-program {
+  display: none;
+}
+.menu_button.show {
+  display: block;
+}
+
+.header-content > a {
+  width: fit-content;
+  text-decoration: none;
+  color: var(--black);
+  font-weight: 800;
+  line-height: 14px;
+  border-bottom: 1px solid var(--medium);
+}
+
+#text_about {
+  text-align: center !important;
+  background-color: var(--white);
+  border: 3px solid white;
+  font-size: 12px;
+}
+
+.text_about {
+  text-align: center !important;
+  font-size: 12px;
+}
+
+.p-box {
+  font-size: 14px;
+}
+
+.hamburguer-menu-icon {
+  display: flex;
+  flex-direction: column;
+  gap: 5px;
+  padding: 5px;
+  cursor: pointer;
+  user-select: none;
+}
+
+.hamburguer-menu-icon > div {
+  background-color: var(--medium);
+  width: 30px;
+  height: 6px;
+  transition: background-color 0.2s linear, transform 0.2s linear;
+}
+
+.desktop-only {
+  display: none;
+}
+
+.t-orange {
+  color: var(--orange);
+  margin-top: 12px;
+}
+
+.texture-text {
+  background: url(Images/orange-bg.png) no-repeat center center;
+  background-size: cover;
+  color: white;
+  -webkit-text-fill-color: transparent;
+  -webkit-background-clip: text;
+}
```

Comment:
- [ Optional]  I will recommend you maximize the font weight and make the title in 2 lines to match the design.